.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

```suggestion
node_modules
.vscode
```
Can also add this to your gitignore :)